cabal.project

```diff
@@ -21,14 +21,6 @@ source-repository-package
     location: https://github.com/tchoutri/pg-entity.git
     tag: e5fc4cf
 
-source-repository-package
-    type: git
-    location: https://github.com/fimad/prometheus-haskell
-    tag: 43f19da
-    subdir: ./prometheus-metrics-ghc
-            ./prometheus-client 
-            ./wai-middleware-prometheus
-
```

Comment:
deleting this made building the nix shell a lot easier.

cabal.project files also tend to bust the nix cache a lot, or rather, it bypasses it entirely forcing many rebuild of libraries. I'm not sure how to make these system play nice together.